x-pack/plugins/security_solution/public/common/components/url_state/helpers.ts

```diff
@@ -23,15 +23,61 @@ import { timelineSelectors } from '../../../timelines/store/timeline';
 import { formatDate } from '../super_date_picker';
 import { NavTab } from '../navigation/types';
 import { CONSTANTS, UrlStateType } from './constants';
-import { ReplaceStateInLocation, KeyUrlState, ValueUrlState } from './types';
+import {
+  ReplaceStateInLocation,
+  KeyUrlState,
+  ValueUrlState,
+  UrlState,
+  UrlStateProps,
+  UrlStateStateToPropsType,
+} from './types';
 import { sourcererSelectors } from '../../store/sourcerer';
 import { SourcererScopeName, SourcererUrlState } from '../../store/sourcerer/model';
+import {
+  CASES_PATH,
+  DETECTIONS_PATH,
+  ALERTS_PATH,
+  RULES_PATH,
+  HOSTS_PATH,
+  NETWORK_PATH,
+} from '../../../../common/constants';
 
 export const isDetectionsPages = (pageName: string) =>
   pageName === SecurityPageName.alerts ||
   pageName === SecurityPageName.rules ||
   pageName === SecurityPageName.exceptions;
 
+// This list provides all the pages that display the security solution alert/events table which
+// launches the flyout when clicking expanded details
+
+const HOST_PAGE_EVENTS_PATH = `${HOSTS_PATH}/events`;
```

Comment:
i think we should consolidate these, define them in common/constants like other urls, then reuse them in places they are used, i.e. https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/hosts/pages/nav_tabs.tsx#L14 and https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/network/pages/navigation/nav_tabs.tsx#L13 or at least use the existing enums for each of these, https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/network/pages/navigation/types.ts#L70 and https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/hosts/store/model.ts#L16 etc

Reply:
Yea, I agree. I'll move these out, and then replace where they exist now in a follow-up PR to not increase the number of files touched here